.travis.yml

```diff
@@ -20,107 +20,36 @@ addons:
       - g++-9
       - g++-10
       - lcov
+      - cmake
+arch:
+  - arm64
+  - s390x
+  - ppc64le
+
+env:
+  - BUILD=coverage    CXX=g++-7  CC=gcc-7  BUILD_TYPE=Debug
```

Comment:
this is neat 🤓

Reply:
Yes, but unfortunately you cannot name it - naming only works for `include`.

But when you use include, no matrix is automatically generated (job X architecture).